### PR TITLE
w3school images lessons

### PR DESCRIPTION
**Pull request by ioandodita069**
astept

### CODE REVIEW

File: html/img.html [hunk 1]
```diff
@@ -5,5 +5,62 @@
...
+
+<h2>Image Size</h2>
+
+<p>Here we specify the width and height of an image with the width and height attributes:</p>
+
+<img src="img_girl.jpg" alt="Girl in a jacket" width="500" height="600">
+
+</body>
+</html><!DOCTYPE html>
+<html>
```
Comment (pancudaniel7) [inline html/img.html:31]: Problem:
- This html file has multiple **html** **body** and **head** tags, one html file should have just one html one body and one head

Comment (ioandodita069) [reply html/img.html:31]: I will resolve this problem

Comment (pancudaniel7) [reply html/img.html:31]: 👍🏻

Comment (pancudaniel7) [reply html/img.html:31]: 👍🏻

File: html/img.html [hunk 2]
```diff
@@ -1,43 +1,30 @@
 <!DOCTYPE html>
 <html>
+  <head>My first try</head>
```
Comment (pancudaniel7) [inline html/img.html:3]: Suggestions:
- this should be align

**Review by pancudaniel7 — APPROVED** [review]
One suggestion but it is looking good, you have my blessings!

**Pull request merged**